docs/claimantmodel/fulcio/identity/logsequence.md

```diff
@@ -0,0 +1,22 @@
+```mermaid
```

Comment:
I hate to be "that guy", but can we add a header to auto-generated files indicating how the file was auto-generated?

Alternatively, maybe none of these files are auto-generated, and the claims and so forth are copied from one to another....

This seems to contain at least some information not contained in other files, so maybe this hand-written.

Reply:
e.g. for generated files (maybe `yaml -> md` if I'm guessing correctly?)

```yaml
<!-- This file generated from $otherfile with:
  $ARGV[*]
-->
```

Reply:
These are generated by the tool with some TODOs. @mhutchinson maybe this would be worth adding a comment like the above in the tool?

Reply:
See https://github.com/google/trillian/pull/3048 for a solution to this.